.eslintrc.yml

```diff
@@ -1,6 +1,8 @@
 env:
   browser: true
   es6: true
+  webextensions: true
```

Comment:
This will prevent eslint from complaining about global objects from the extension environment (like `browser`).